wdl-doc/src/callable/workflow.rs

```diff
@@ -0,0 +1,136 @@
+//! Create HTML documentation for WDL workflows.
+
+use maud::Markup;
+use wdl_ast::AstToken;
+use wdl_ast::v1::InputSection;
+use wdl_ast::v1::MetadataSection;
+use wdl_ast::v1::MetadataValue;
+use wdl_ast::v1::OutputSection;
+use wdl_ast::v1::ParameterMetadataSection;
+
+use super::*;
+use crate::meta::render_value;
+use crate::parameter::Parameter;
+
+/// A workflow in a WDL document.
+#[derive(Debug)]
+pub struct Workflow {
+    /// The name of the workflow.
+    name: String,
+    /// The meta of the workflow.
+    meta: MetaMap,
+    /// The inputs of the workflow.
+    inputs: Vec<Parameter>,
+    /// The outputs of the workflow.
+    outputs: Vec<Parameter>,
+}
+
+impl Workflow {
+    /// Create a new workflow.
+    pub fn new(
+        name: String,
+        meta_section: Option<MetadataSection>,
+        parameter_meta: Option<ParameterMetadataSection>,
+        input_section: Option<InputSection>,
+        output_section: Option<OutputSection>,
+    ) -> Self {
+        let meta = if let Some(ref mds) = meta_section {
+            parse_meta(mds)
+        } else {
+            MetaMap::default()
+        };
```

Comment:
Same comment as above re: `let` usage:

```suggestion
        let meta = if let Some(mds) = meta_section {
            parse_meta(&mds)
        } else {
            MetaMap::default()
        };
```